spec/features/user_visits_place_spec.rb

```diff
@@ -56,4 +74,22 @@
       end
     end
   end
+
+  describe "that doesn't exist" do
+
+    it 'renders a SORRY page' do
+      VCR.use_cassette("nonexistnat place") do
+        visit '/'
+
+        page.fill_in 'nav-search',
+          :with => 'askljasdf'
+
+        click_button "Go"
+
+        expect(current_path).to eq('/')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.